### PR TITLE
Use one database for tests

### PR DESCRIPTION
**Pull request by emord**
opening up for travis tests. Had an idea that I hope might speed up tests?

### CODE REVIEW

**Comment by emord** [comment]
good news: It appears that by using an `OrderedDict` it got past the issue we were seeing where a couch view was not created

bad news: It looks like the AAA migrations were not applied correctly (table aaa_child doesn't exist) potentially still a good avenue to go down to reduce test times

**Comment by snopoke** [comment]
@emord can you elaborate on the couch view issue?

**Comment by snopoke** [comment]
If DB order is important you can do that by specifying dependencies: https://docs.djangoproject.com/en/2.1/topics/testing/advanced/#controlling-creation-order-for-test-databases

**Comment by emord** [comment]
@snopoke https://github.com/dimagi/commcare-hq/pull/23351#issuecomment-467500691

> Right now both default and icds-ucr resolve to the database commcarehq when I added aaa-data the couch views started faililng. I have no idea why that was, but my best guess is a timing issue with our sync couch views and migration commands.

**Review by millerdev — APPROVED** [review]
Looking good, hope you can get this to work.

**Review by nickpell — APPROVED** [review]
I wonder if these will pass if the 4->5 test jobs change is reverted.

**Comment by emord** [comment]
@nickpell it do

```
corehq.apps.userreports.tests.test_report_builder:ReportBuilderTest () ... SKIP: divided-we-run: 3 not in range bf
corehq.apps.userreports.tests.test_report_config:ReportConfigurationDbTest.test_config_id_is_required ... ok
corehq.apps.userreports.tests.test_report_config:ReportConfigurationDbTest.test_domain_is_required ... ok
corehq.apps.userreports.tests.test_report_config:ReportConfigurationDbTest.test_get_all ... ok
corehq.apps.userreports.tests.test_report_config:ReportConfigurationDbTest.test_get_by_domain ... ok
corehq.apps.userreports.tests.test_report_config:ReportConfigurationDbTest.test_sample_config_is_valid ... ok
```

not sure why but it appears that the AAA aggregation tasks hang at the end of the test run. tried adding a patch to the teardown method but my local setup is now different than what's on travis so i'll have to attempt debugging that tomorrow

File: testsettings.py [hunk 1]
```diff
@@ -117,36 +120,10 @@ def _set_logging_levels(levels):
...
-        'NAME': 'aaa_commcarehq',
-        'USER': 'commcarehq',
-        'PASSWORD': 'commcarehq',
-        'HOST': 'localhost',
-        'PORT': '5432',
-        'TEST': {
-            'SERIALIZE': False,
-        }
-    }
+    DATABASES['icds-ucr'] = deepcopy(DATABASES['default'])
```
Comment (calellowitz) [inline testsettings.py]: Why is ICDS copied but aaa not?

**Review by calellowitz — APPROVED** [review]
Looks great. I can't believe you merged those migrations so fast. this should be a huge win

**Pull request closed without merging**